### DISCUSSION TITLE
What is the relationship between DataspaceConnector and Trusted-connector

### DISCUSSION THREAD

**Discussion by tanchy82** [International-Data-Spaces-Association/DataspaceConnector#117] [Q&A] [answered]
What is the relationship between DataspaceConnector and Trusted-connector  ?

-    https://github.com/International-Data-Spaces-Association/DataspaceConnector
-    https://github.com/International-Data-Spaces-Association/trusted-connector
I want to achieve a IDS connector, but i can see two connector in the International-Data-Spaces-Association github repos, I don't know how to choose？

**Comment by HeinrichPet** [comment, marked as answer]
There are even more than these two.
If you want to start with little effort, use the Dataspace Connector.
However, the Dataspace Connector is no longer being developed, so the future-proof and scalable variant is the Eclipse Dataspace Connector: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector

**Reply by tanchy82** [reply]
Thank you very much, HeinrichPet. 
I would like to start with the operational use  Dataspace Connector.
Would you like a copy of the Dataspace Connector manual with examples of how to use it that you can help with? 
Thank you！

**Comment by MeiYiWZz** [comment]
Hello，Have you implemented the IDS connector yet?
